## Methods

### Multifactor Dimensionality Reduction (MDR) and model-based MDR (MB-MDR)

MDR is a nonparametric method that detects multiple genetic loci associated with a clinical outcome by reducing the dimension of a genotype dataset by pooling multilocus genotypes into high-risk and low-risk groups [@doi:10.1086/321276].
Extended from the original MDR algorithm, MB-MDR addresses existing limitations of MDR by increasing detectability of important interactions and decreasing bias by allowing O labels for individuals with no evidence for abberant risk.
Several improvements have been made to MB-MDR since it was first introduced in 2009, and its current implementation efficiently and effectively detects multiple sets of significant gene-gene interactions in relation to a trait of interest while efficiently controlling type I error rates.
Besides the P values associated with each genotype combination, another important output of MB-MDR is the HLO matrices generated from the affected- and unaffected-subjects matrices (in the case of binary outcome).
Briefly, for each genotype combination, an HLO matrix is a 3$\times$3 matrix with each cell containing H (high), L (low) or O (no evidence), indicating risk of an individual whose genotype pairs fall into that cell [@doi:10.1186/1471-2105-14-138].
For an example binary outcome problem, a genotype combination $SNP_1$ and $SNP_2$ will have a $\chi^2$ value, an associated P value and an HLO matrix that looks like
$$ \begin{array}{l|ccc}
& SNP_1 = 0 & SNP_1 = 1 & SNP_1 = 2   \\
\hline
SNP_2 = 0 & O        & O        & O \\
SNP_2 = 1 & O        & H        & L \\
SNP_2 = 2 & O        & L        & H
\end{array}.
$$
We discuss in the following subsection how these values were utilized in the formulation of the Multilocus Risk Score (MRS).

### Multilocus Risk Score (MRS)
We apply the MB-MDR software [@doi:10.1186/1471-2105-14-138] v.4.4.1 to simulated datasets of $n$ individuals, $p$ SNPs to obtain the significance level of each combination of SNPs and let $k_d$ denote the number of significant combinations.
In this study, no significance threshold is imposed and thus $k_d$ reaches its maximum value of $C^d_p$.

For each subject $i$ ($i = 1,2,\cdot, n$), the $d$-way interaction risk score is calculated as
$$MRS_d(i) = \sum_{j = 1}^{k_d} \chi_j^2 \times \textrm{HLO}_j(X_{ij})$$
where $\chi_j^2$ is the test statistic of each genotype combination $j$ from a $\chi_j^2$ test with one degree of freedom for the simulated binary trait, $X_{ij}$ is the $j^{th}$ genotype combinations of subject $i$ and $\textrm{HLO}_j$ represents the $j^{th}$ recoded HLO matrix (1 = High, -1 = Low, 0 = No evidence).
As an example, consider a pair $X_{*1} = (SNP_1, SNP_2)$ with $\chi_j^2=8.3$ and corresponding HLO matrix of all O's except an L, or -1, in the first cell.
Then, all subjects' current risks would remain the same except the ones with $SNP_1 = SNP_1 = 0$ where their risks are subtracted by 8.3.

 
The final MRS score is the sum of all $MRS_d$ for all $d$ up to $\bar{d}$:
$$MRS(i) = \sum_{d = 1}^{\bar{d}} MRS_d(i)$$
In this study, we consider 1-way and 2-way interactions, i.e. $\bar{d} = 2$, and hence, the combined risk is simply the total of the first two: $MRS = MRS_1 + MRS_2$.


### Simulated data
[Patryk...]

[objective: simulate a diverse collection of datasets]

For each simulated and real-world dataset, after randomly splitting the entire data in two smaller sets (80% training and 20% holdout), we built the MRS model on training data to obtain the $\chi^2$ coefficients and calculated risk score for each individual in the holdout set.
We assess the performance of the MRS by comparing the area under the Receiving Operator Characteristic curve (auROC) with that of the standard GRS method.

### Mutual information and information gain
$I(G_1,y)$ can be calculated based on Shannon's entropy [@doi:10.1002/j.1538-7305.1948.tb01338.x] and can be used as a measure of the main effect of the genotype $G_j$ on the phenotypic class $y$.

 entropy-based methods to measure how much information about the endpoint is due to the synergistic effects of the variants after subtracting the marginal effects
 
We measure the 2-way interaction information (i.e. degree of synergistic effects of genotypes on the phenotype) of each dataset by summing the pairwise information gain between all pairs of genetic attributes.
Specifically, given a dataset with phenotypic class $y$, its total 2-way interaction gain is calculated as ...
$$II = \sum_{j}^kIG(X_j, C) $$
where $IG$ mesures how much of the phenotypic class can be explained by the epistatic interaction between two genotypes) represents the mutual information between two attributes.

two-way epistatic interactions


