## Methods

### Multifactor Dimensionality Reduction (MDR) and model-based MDR (MB-MDR)

MB-MDR is a feature selection method that detects multiple sets of significant gene-gene interactions in relation to a trait of interest, while efficiently controlling type I error rates.


### Multilocus Risk Score (MRS)
Compute risks from significant interactions
i = 1...n subjects
p SNPs
j = 1...k 
We apply the MB-MDR software [@doi:10.1186/1471-2105-14-138] to obtain the significance level of each combination of SNPs.
[allow for parallel computation]
If we let $k$ be the number of significant combinations then, for each subject $i$, the $d$-way interaction risk score is calculated as
$$MRS_d(i) = \sum_{j = 1}^k \chi_j^2 \times HLO_j(X_{ij})$$
where $\chi_j^2$ is the test statistic of each genotype combination $j$ from a $\chi_j^2$ test with one degree of freedom for the simulated binary trait, $X_{ij}$ is the $j^{th}$ genotype combinations of subject $i$ and $HLO_j$ represents the $j^{th}$ re-coded HLO-matrix (1 = High, -1 = Low, 0 = No evidence).
For example, if [...].
The maximum value of $k$ is $C^d_p$ (when no threshold is imposed).
 
The 
$$MRS(i) = \sum_{d = 1}^{\bar{d}} MRS_d(i)$$
In this study, we consider 1-way and 2-way interactions, i.e. $\bar{d} = 2$, and hence, the combined risk is simply the total of the first two: $MRS = MRS_1 + MRS_2$.


### Simulated data
[Patryk...]

[objective: simulate a diverse collection of datasets]

For each simulated and real-world dataset, after randomly splitting the entire data in two smaller sets (80% training and 20% holdout), we built the MRS model on training data to obtain the $\chi^2$ coefficients and calculated risk score for each individual in the holdout set.
We assess the performance of the MRS by comparing the area under the Receiving Operator Characteristic curve (auROC) with that of the standard GRS method where

### Mutual information and information gain
We measure the interaction information (i.e. degree of synergistic effects of genetic attributes on the phenotype) of each dataset by summing up the pairwise information gain between all pairs of genetic attributes.
Specifically,... phenotypic class $C$...
$$IG(G_1, G_2; C) = I(G_1, G_2; C) - I(G_1;C) - I(G_2,C),$$
where $I$ represents the mutual information between two attributes.

two-way epistatic interactions


