## Discussion
PRS holds much promise for development of new precision medicine approaches by identifying high risk individuals who may benefit from prioritized interventions.
Nevetheless, one of its current limitation is the model simplicity [@doi:10.1038/s41576-018-0018-x].
As a first step towards addressing this issue and increasing comprehensiveness of risk profiling models, in this study, we developed a new model that enables automatic genotype encodings and takes into account gene-gene interactions.
Utilizing the efficient implementation of MB-MDR, MRS automatically infers the genotype encodings and simultaneously computes the risk matrices of pairs of variants.

We showed that the MRS method outperformed standard PRS in many of the simulated datasets, highlighting the importance of genotype encodings and consideration of epistasis.
We further examined the association between this improvement and the amount of two-way epistatic effect induced in the binary phenotypic outcome.
Appropriate phenotype encodings are important for improving the accuracy when there is a large amount of main effects of the variants on the phenotypic outcome.
Meanwhile, inclusion of epistatic terms significantly increases the accuracy from PRS, especially when two-way interactions are present in the data.
Although we only considered up to two-way interactions, it is straightforward to incorporate higher order interactions (e.g. three-way, four-way) into MRS.
However, preliminary analyses on the simulated datasets did not show much improvement from the current MRS (results not shown).
We also recommend estimating the computational expense prior to implementing high order interactions, especially for larger datasets encountered in practice.

We acknowledge a few limitations of the current study.
First, MRS has not been applied to real-world data.
Although we compensated the lack of real data with a diverse, unbiased set of simulated datasets, a future study analyzing will prove beneficial to quantify the new MRS model's utility in practice.
Second, accounting for epistasis, in principle, is largely more computationally expensive compared to investigating solely main effects.
Therefore, even with fast and efficient softwares, pre-selecting the variants (e.g. based on specific pathways or prior knowledge) will prove beneficial for accurate MRS computing when analyzing datasets containing a larger number of variants.
However, we hope the promising preliminary results will open the doors to future approaches that encompass main and interaction effects and improve scalability.

Finally, we caution that a risk score model should be evaluated based on not only sensitivity and specificity but also with respect to potential clinical efficacy, and any genetic risk should be interpreted in aggregate with other risk factors.
Future works focusing on gene-environment interactions with time-dependent risk factors will be crucial in order to communicate risk properly for preventive interventions.

In conclusion, MRS enhances the predictive capacity of current risk profiling model for complex diseases with polygenic architectures.
While there is much work left to do to improve the personal and clinical utility of general risk profiling framework, we highlight that more comprehensive models that infer proper genotype encodings and account for epistatic effects greatly improve the prediction efficiency and affords new opportunities for more accurate clinical prevention.